ckanext/notify/templates/notify/channels.html

```diff
@@ -35,11 +35,11 @@ <h2 class="page-heading">{% trans %}DataRequest Notification Channel{% endtrans
               {% set locale = h.dump_json({'content': _('Are you sure you want to delete this notification channel?')}) %}
               <div class="btn-group pull-right">
                 <a class="btn btn-small" href="{{ h.url_for('update_slack_form', id=channel.id, organization_id=channel.organization_id) }}" title="{{ _('Edit') }}">
-                  <i class="icon-wrench"></i>
+                  <i class="fa fa-wrench"></i>
```

Comment:
Is this commit realy supposed to be under the task, add -travis, I do not seem to see the connection between the work done here and adding  travis, t how about always doing a specific related unit of work into one atomic commit